backend/src/main/java/me/quizzl/backend/services/QuizService.java

```diff
@@ -19,6 +20,10 @@ public Quiz addQuiz() {
         return quizRepository.save(quiz);
     }
 
+    public Quiz getQuizByID(UUID id){
+        return quizRepository.getOne(id);
```

Comment:
my bad, this should probably return `quizRepository.findById(id)` instead, so the application doesn't throw if no quiz by that ID is found

Reply:
`findById` returns an `Optional<Quiz>` instead of a `Quiz`, so you'll have to change the type that this method returns to match btw. that's why the build's failing right now